types/store/Store.raml

```diff
@@ -0,0 +1,23 @@
+#%RAML 1.0 DataType
+(package): Store
+(docs-uri): https://docs.commercetools.com/http-api-projects-stores#stores
+
+type: object
```

Comment:
I know its already merged, but shouldn't this be a `Resource` type instead of an `Object` (and remove properties `id`, `version`, `createdAt`, `lastModifiedAt` from this file)?